remove dev dependencies

jimmywarting opened the pull request:
use built in instead.

jimmywarting committed:
remove dev dependencies

jimmywarting committed:
update node v

JonahGroendal commented:
I haven't looked at this project in forever. Do the tests pass and does it still work in the browser? That's the main point of the polyfills as I recall. Are you using it in the browser or a nodejs environment?

jimmywarting commented:
I only touched the test files. Nothing affected the browser or the code. I just updated node version, and removed node fetch and btoa. Which is now built in to node

JonahGroendal commented:
thanks for the PR @jimmywarting

JonahGroendal commented:
@jimmywarting this broke the tests actually. They were working just before I merged the PR (well they were getting farther anyway. Two of the tests were passing now none are)

https://github.com/JonahGroendal/acme-easy/actions/runs/15378025686/job/43265188805?pr=19

Want to submit another PR to fix it or should I roll back the change? You'll need to work off of branch fix-workflow where I fixed the github workflow
